socorro/unittest/processor/test_processor_2015.py

```diff
@@ -4,123 +4,153 @@
 from configman.dotdict import DotDict
 from mock import Mock, patch
 
-from socorro.lib.transform_rules import TransformRuleSystem
-from socorro.processor.processor_2015 import (
-    Processor2015,
-    rule_sets_from_string
-)
+from socorro.processor.processor_2015 import Processor2015
 from socorro.processor.general_transform_rules import (
     CPUInfoRule,
     OSInfoRule
 )
 from socorro.unittest.testbase import TestCase
 
 
-rule_set_01 = [
-    [
-        'ruleset01',
-        'tag0.tag1',
-        'socorro.lib.transform_rules.TransformRuleSystem',
-        'apply_all_rules',
-        'socorro.processor.general_transform_rules.CPUInfoRule, '
-        'socorro.processor.general_transform_rules.OSInfoRule '
-    ]
-]
-rule_set_01_str = ujson.dumps(rule_set_01)
-
-
 class TestProcessor2015(TestCase):
-    def test_rule_sets_from_string_1(self):
-        rule_set_config = rule_sets_from_string(rule_set_01_str)
-        rc = rule_set_config.get_required_config()
-        assert 'ruleset01' in rc
-        assert 'tag0.tag1' == rc.ruleset01.tag.default
-        expected = 'socorro.lib.transform_rules.TransformRuleSystem'
-        assert rc.ruleset01.rule_system_class.default == expected
-        assert 'apply_all_rules' == rc.ruleset01.action.default
-        expected = (
-            'socorro.processor.general_transform_rules.CPUInfoRule, '
-            'socorro.processor.general_transform_rules.OSInfoRule '
-        )
-        assert rc.ruleset01.rules_list.default == expected
 
     def test_Processor2015_init(self):
         cm = ConfigurationManager(
             definition_source=Processor2015.get_required_config(),
-            values_source_list=[{'rule_sets': rule_set_01_str}],
+            values_source_list=[],
         )
         config = cm.get_config()
         config.logger = Mock()
 
-        p = Processor2015(config)
+        Processor2015(config)
```

Comment:
I'm not sure I understand what this is testing. Is it testing that we can initialize a `Processor2015` in general? If so, we can probably just nix it since we do that inherently in other tests.

Reply:
Yep, as the name implies it's only testing that the processor initializes correctly. This was useful when moving configuration around. Rules missing config would break loudly here. Could take it or leave it now that initialization is simplified and fully redundant with the other rules.